trieste/models/model_interfaces.py

```diff
@@ -504,29 +504,47 @@ class SparseVariational(GPflowPredictor, TrainableProbabilisticModel):
     A :class:`TrainableProbabilisticModel` wrapper for a GPflow :class:`~gpflow.models.SVGP`.
     """
 
-    def __init__(self, model: SVGP, data: Dataset, optimizer: Optimizer | None = None):
+    def __init__(
+        self,
+        model: SVGP,
+        optimizer: Optimizer | None = None
+    ):
         """
         :param model: The underlying GPflow sparse variational model.
-        :param data: The initial training data.
         :param optimizer: The optimizer with which to train the model. Defaults to
-            :class:`~trieste.models.optimizer.Optimizer` with :class:`~gpflow.optimizers.Scipy`.
+            :class:`~trieste.models.optimizer.Optimizer` with :class:`~tf.optimizers.Adam`.
+        :param max_num_inducing_points: The maximum number of inducing points allowed.
```

Comment:
`max_num_inducing_points` is not present as an argument in `__init__` above?

Reply:
Thanks! I initially tried to add this as an option but decided to just get a basic interface working instead and add that later, but forgot to remove the relevant line.